### PR TITLE
Fix link to TypeScript type definitions

### PR DESCRIPTION
**Pull request by mbtools**

### CODE REVIEW

File: website/docs/dev-plugins.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ There are many ways to extend `verdaccio`, the kind of plugins supported are:
 * Theme
 * Filter plugins
 
-> We recommend developing plugins using our [Typescript type definitions](https://github.com/verdaccio/monorepo/tree/master/core/types).
+> We recommend developing plugins using our [TypeScript type definitions](https://github.com/verdaccio/verdaccio/tree/master/packages/types).
```
Comment (juanpicado) [inline website/docs/dev-plugins.md:14]: Actually, the website still points to v5, so this link still correct, until I merge https://github.com/verdaccio/verdaccio/pull/3320 probably we still have to maintain like that so I can finally start to modify docs for v6 with master content.

Comment (mbtools) [reply website/docs/dev-plugins.md:14]: The current link on https://verdaccio.org/docs/dev-plugins leads to a 404. Hmmm

Comment (juanpicado) [reply website/docs/dev-plugins.md:14]: Works for me that link.

Comment (mbtools) [reply website/docs/dev-plugins.md:14]: that's odd:

chrome_738

https://github.com/verdaccio/monorepo/tree/master/core/types > 404
(core/types exists in the 9.x branch but not in master)

Comment (juanpicado) [reply website/docs/dev-plugins.md:14]: Ahhh, you refer to that link, it is not master anymore, is `9.x` <- don't ask me why :D is 9.x 

https://github.com/verdaccio/monorepo/tree/9.x/core/types

so the right link should be that one.

**Review by juanpicado — APPROVED** [review]

**Pull request merged**